bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -127,8 +127,12 @@
           {% if release.product == 'Firefox for Android' %}
           <li class="try3"><a href="https://support.mozilla.org/products/mobile/download-and-install">{{ _('Install') }}</a></li>
           {% else %}
+          {% if release.product == 'Firefox for iOS' %}
+          <li class="try3"><a href="https://support.mozilla.org/products/ios/install-and-update-firefox-ios">{{ _('Install') }}</a></li>
+          {% else %}
           <li class="try3"><a href="https://support.mozilla.org/products/firefox/download-and-install">{{ _('Install') }}</a></li>
           {% endif %}
+          {% endif %}
           <li class="try4"><a href="https://addons.mozilla.org/firefox/">{{ _('Customize') }}</a></li>
```

Comment:
Standard addons will not work with Firefox for iOS, so we should remove this link.